### ISSUE TITLE
Install instructions or script ?

### ISSUE THREAD

**Issue by jppaolim** [knoopx/llm-workbench#1]
Looks Amazing, but ... I don't see the right node script anywhere. How do you run this ? If I just start ollama, I have a great server but what do I do with it ? Thanks !!

**Comment by jppaolim** [comment]
btw, I am on MacOS, I have cloned the repo, run a "npm install" there and  now I am stuck...

**Comment by knoopx** [comment]
There's really nothing to install (just ollama or any public or local HF TGI endpoint), just vist https://knoopx.github.io/llm-workbench/ and configure your agents, settings/history are persisted in the browser local storage. The web app directly connects to your endpoints, no middle-wares between. If you want to self host, it's a regular vite app, run `npm run vite build` and will generate a `dist` folder with all the static app files (you will need a web server to serve them). You can alternatively run `npm run vite` to start a development server.

**Comment by jppaolim** [comment]
Thanks. Now I get it. It wasn't clear for me why I had to put OLAMA_ORIGIN in the parameter. Still when I go to the page you mention, once I click chat , I have a blank page.  Then I try to local host because anyway, I am not confortable with sending stuff online...  but npm run vite build doesn't do anything (there is no script "vite") I did NPX VITE which starts something but then it becomes the limit of my understanding with vite... 
Bottom line:  looks really great and one of the few app (if not the only one to date) with "agent" concept embedded, but for now, quite hard to start using it...which is the opposite of ollama philosophy. So if you or somebody could PR a release of some install script, that would be amazing. Good luck !

**Comment by knoopx** [comment]
Hey @jppaolim the blank page was due to a bug. Fixed it already.

**Comment by jppaolim** [comment]
amazing , just works !  a bit of polish still to do but amazing interface / work and one of the few that is taking on the concept of agents. One comments (but might not be the place to ask, what happen when you attach a file ?)

**Comment by knoopx** [comment]
@jppaolim attachments are processed as text and passed by as the variable `attachments` and can be  used in the prompt template to be injected in the context for QA agents. I've been playing with ollama  and transformers.js embeddings for proper RAG but they both are too slow when processing large documents.
